src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -76,7 +76,7 @@ used with signed exchanges.
    request](https://en.wikipedia.org/wiki/Certificate_signing_request) (CSR).
 
     ```shell
-    openssl req -new -sha256 -key priv.key -out cert.csr -subj '/O=Web Packager Demo /CN=example.com'
+    openssl req -new -sha256 -key priv.key -out cert.csr -subj '/O=Web Packager Demo/CN=example.com'
```

Comment:
Looks like the default value in the toml for `Sign.Domain` is 'example.org' ([src](https://github.com/google/webpackager/blob/d337403e9772ff01b930c6084918bd583a914cb1/cmd/webpkgserver/webpkgserver.example.toml#L308)). We should either:

 1. Change `example.com` to `example.org` throughout this doc.
 2. Include a step telling people to change the Sign.Domain in the toml to `example.org`.
 3. Update the example TOML to match.

I'm open to any of these.

Reply:
Oops, sorry, I see you did (2) below. Resolving.